sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -2404,7 +2423,10 @@ pi_result piProgramGetInfo(pi_program Program, pi_program_info ParamName,
       // in Level Zero.  Therefore, this API is unimplemented when the Program
       // has more than one module.
       _pi_program::ModuleIterator ModIt(Program);
-      assert(!ModIt.Done());
+      if (ModIt.Done()) {
+        return PI_INVALID_VALUE;
```

Comment:
what does it mean? should we die or expect RT to handle that?

Reply:
programs in LinkedExe state can have multiple modules and ModIt is an iterator to access those modules. Done() always returns false for object and exe. for LinkedExe it checks if the iterator is empty. As this is a query, I think I'll just do a PI_ASSERT here.

Reply:
OK, thanks